src/wallet/wallet.h

```diff
@@ -24,6 +24,7 @@
 #include "wallet/walletdb.h"
 #include "wallet/rpcwallet.h"
 #include "zcash/address/bip44.h"
+#include "zcash/address/unified.h"
```

Comment:
Why is this import addition necessary in the second commit?

Reply:
`unified.h` is removed from the dependencies of `Address.hpp`, so it is no longer transitively available here.